loadtest/control/simulcontroller/actions.go

```diff
@@ -2007,6 +2007,22 @@ func (c *SimulController) openUserProfile(u user.User) control.UserActionRespons
 	if err := u.GetChannelMember(ch.Id, post.UserId); err != nil {
 		return control.UserActionResponse{Err: control.NewUserError(err)}
 	}
+
+	cpaEnabled, resp := control.CustomProfileAttributesEnabled(u)
+	if resp.Err != nil {
+		return resp
+	}
+	if cpaEnabled {
+		// attempt to retrieve from store
+		attributes := u.Store().GetCPAValues(post.UserId)
+		if len(attributes) == 0 {
+			// Retrieve custom profile attribute values for the user.
+			if attributes, err = u.GetCPAValues(post.UserId); err != nil {
+				return control.UserActionResponse{Err: control.NewUserError(err)}
+			}
+		}
+	}
```

Comment:
This works just as it does in the webapp. Attempts to retrieve from redux, if they haven't been retrieved, retrieve them from server.